src/main/webapp/scripts/tests/comment-class-test.js

```diff
@@ -0,0 +1,27 @@
+QUnit.test('Comment Class - getError (normal use)', function(assert) {
```

Comment:
nit: If QUnit supports that, is it possible to move the test files to src/test/webapp/script/...? That way test code and main code can be separated

Reply:
Yes, but it would result in some tags like "../../../test/webapp/script/" because the tests require both js files in the script directory and test directory. Might be easy to get confused with all the ../, what do you think?

The other option is to use an absolute file path "/src/test/webapp/script/.." and "/src/webapp/script" but I read somewhere that relative file paths are preferred.

Reply:
I always thought absolute paths is preferred lol. It's okay, not a big deal. No need to change here. 

Does Qunit run automatically after `mvn appengine:run` is called?

Reply:
Hmm googled it and found some mixed answers, the style guide tends to use relative paths though, the main reason I found for relative paths is that its easier to change file names but I don't think we'll be needing to do that. as long as we stick with one way it should be fine

Qunit runs when you load the website at /test.html, I guess from now on we'll have to check that page before every push. Me and Kaye did not find a JS testing framework that was convenient to use with mvn test :/

Reply:
these are ways to connect them all (qunit or other testing frame works, node and mvn) but it seems like a pain from what I have found (this is probably what cicd folks do?). I agree right now we can just manually check the test page.

Reply:
You're right, I just searched and found a couple options but some issues I encountered were
-could not find documentation
-licenses
-broken links
Probably better to just use the test page and look into this again later if needed. Or if you find one that might work, link me and I'll look into it!